docs/src/main/sphinx/connector/hive.rst

```diff
@@ -721,6 +721,32 @@ The connector supports the following commands for use with
     to the Trino logs and query failure messages to see which files need to be
     deleted.
 
+SQL security enforcement
```

Comment:
This information belongs in Hive security doc at the same level as "SQL standard based authorization". Other types of hive.security should be documented there too (not required in this change itself).

This page can maybe just mention that Hive connector supports Connector access control and that the default is legacy and link them to proper place?

WDYT @jhlodin ?

Reply:
My concern is that an outcome of current behavior is a user runs a statement like `DROP TABLE` and is told in the UI that it's unsupported, then in this SQL support section we say it's supported. This new content can be more succinct than this, but I do think we need a way to catch those readers who won't make the security connection.

The "SQL security enforcement" header could easily be changed to better reflect this content though. Maybe "Manage SQL statement support" or similar?

Reply:
BTW the error they get isn't that DROP TABLE isn't supported or invalid syntax. They get an Access Denied error mentioning that they don't have the privilege.

Reply:
That's a good idea, and very glad to hear that the error message is more descriptive. Let me try that